metadata/org.liquibase/liquibase-core/4.23.0/reflect-config.json

```diff
@@ -0,0 +1,1256 @@
+[
```

Comment:
Hey, how did you generate these new entries when there is no new tests provided?

Reply:
I used the existing test created for `4.17.0`. The same thing was done for `4.20.0` and now for `4.23.0`. The github job executes the same test three times (each time with different metadata version).